@xen-orchestra/web/src/components/host/HostHeader.vue

```diff
@@ -35,7 +35,7 @@
     <TabItem disabled>{{ t('tasks') }}</TabItem>
     <RouterLink v-slot="{ isActive, href }" :to="`/host/${host.id}/vms`" custom>
       <TabItem :active="isActive" :href tag="a">
-        {{ t('vms') }}
+        {{ t('vms', 0) }}
```

Comment:
According to Figma, show the vms variant.
```suggestion
      {{ t('vms', 2) }}
```